app/resources/static/home.html.template:ja

```diff
@@ -1,82 +0,0 @@
-{% extends "static-base.html.template" %}
```

Comment:
Why did you delete these files?

Reply:
Will fix, i think the branch i checked out from must have been different from master.